charts/kubewarden-defaults/README.md

```diff
@@ -20,20 +20,22 @@ user must enable this feature.*** The recommended policies are:
 
 All the policies are installed cluster wide. But they are configured to ignore
 namespaces important to run the control plane and Rancher components, like
-`kube-system` and `rancher-operator-system` namespaces.
+`kube-system` and `rancher-operator-system` namespaces. This chart provides a list
+of namespaces that are not included in the scanning, which can be found [here](https://github.com/kubewarden/helm-charts/blob/main/charts/kubewarden-defaults/values.yaml#L77)
```

Comment:
How about rewording it to:

```
All the policies are installed cluster wide. But they are configured to ignore
namespaces important to run the control plane and Rancher components, like
`kube-system` and `rancher-operator-system` namespaces. This list of default
ignored namespaces is in the chart values under
`recommendedPolicies.skipNamespaces`.
```
(I don't think we should link to a github path, or a line of values.yaml, both can change)

Reply:
Added the requested change to the README

Reply:
Thanks! merging